docs-web/configuration/virtualserver-and-virtualserverroute-resources.md

```diff
@@ -186,6 +189,10 @@ The route defines rules for matching client requests to actions like passing a r
      - The name of a VirtualServerRoute resource that defines this route. If the VirtualServerRoute belongs to a different namespace than the VirtualServer, you need to include the namespace. For example, ``tea-namespace/tea``.
      - ``string``
      - No*
+   * - ``errorPages``
```

Comment:
we need to add the same for the subroute

Reply:
Oops, good catch